tests/webTables.spec.ts

```diff
@@ -55,10 +55,10 @@ test.describe('Web Tables', () => {
     test('Test Case 4: Validate phone number and pet name on the Owner Information page ', async ({ page }) => {
         await page.getByRole('button', { name: 'Owners' }).click()
         await page.getByRole('link', { name: 'Search' }).click()
-        const targetRow = page.getByRole('row', { name: '6085552765' })
-        const petName = await targetRow.locator('td').nth(4).textContent()
-        await targetRow.getByRole('link', { name: 'Peter McTavish' }).click()
-        await expect(page.getByRole('cell', { name: '6085552765' })).toContainText('6085552765')
+        const phoneNumber = '6085552765'
+        const petName = await page.getByRole('row', { name: '2387 S. Fair Way' }).locator('td').nth(4).textContent()
+        await page.getByRole('row', { name: '2387 S. Fair Way' }).getByRole('link', { name: 'Peter McTavish' }).click()
```

Comment:
This locator can be simplified, because the name of the owner is unique
`await page.getByRole('link', { name: 'Peter McTavish' }).click()`